planning/retrospective.md

```diff
@@ -35,4 +35,46 @@ This retrospectives will help our team to improve our collabration for the futur
 - The benefits of communication
 - Having team-leader is well for organisation and planning
 
-![retro](https://github.com/yildiztugba/HYKWebsite/blob/main/planning/RetroHYF1.jpg)
+
+
+
+# Retrospective (Week 3 26.04 - 02.05) 🤟
+
+
+## Stop Doing
+
+- Save unformatted code before sending PR
```

Comment:
These Two lines is wıth different format than others, can you check them?

And also it seems like week 1 and week 2 are different in format. Do you want to change?